config/optional/block.block.oe_theme_corporate_footer.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - oe_corporate_blocks
+  theme:
+    - oe_theme
+id: oe_theme_corporate_footer
+theme: oe_theme
+region: footer
+weight: 0
+provider: null
+plugin: oe_footer
+settings:
+  id: oe_footer
+  label: 'OpenEuropa Footer block'
```

Comment:
Let's just use "Corporate footer" as label, we received instructions from COMM to try not to have OpenEuropa visible on UIs.

Reply:
fixed